### PR TITLE
added github actions

### PR DESCRIPTION
**Pull request by silentwraith6**
added github actions #15

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Dolos CI/CD
+
+on: [pull_request, issues, push]
```
Comment (YashKumarVerma) [inline .github/workflows/actions.yml:3]: do we need to add the entry named `issue` there ?

also, please sync your pr with the master branch.

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

File: .github/workflows/actions.yml (hunk 1 above)
Comment (silentwraith6) [reply .github/workflows/actions.yml]: removed issue entry point. Please check!
Also my PR synced with master branch

File: package.json [hunk 2]
```diff
@@ -30,6 +30,14 @@
...
+    "jest": "^26.4.2",
+    "nodemon": "^2.0.4",
+    "supertest": "^5.0.0"
+  },
+  "jest": {
+    "testEnvironment": "node",
+    "coveragePathIgnorePatterns": [
+      "/node_modules/"
+    ]
   }
```
Comment (YashKumarVerma) [inline package.json:42]: isn't this change already in the repository ?
image

Comment (silentwraith6) [reply package.json:42]: I am little bit confused, should I just fork it again as a fresh one?

**Comment by YashKumarVerma** [comment]
Marking duplicate of #22

**Pull request closed without merging**